webapp/channels/src/components/interactive_dialog/interactive_dialog.test.tsx

```diff
@@ -123,7 +123,7 @@ describe('components/interactive_dialog/InteractiveDialog', () => {
         const props = {
             ...rest,
             elements: [
-                ...elements,
+                ...(elements as []),
```

Comment:
Why this is `as []`, it can be as simple as `|| []`, that will be still a list and would mess with the type system.